.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
     docker:
       # specify the version you desire here
-      - image: circleci/openjdk:8-jdk
+      - image: circleci/openjdk:11.0
```

Comment:
@schowsf probably better to use the later convenience image, i.e. `cimg/openjdk:11.0`